maintenance/MaintenanceReferenceOntology.rdf

```diff
@@ -477,19 +477,19 @@
 		<owl:equivalentClass>
 			<owl:Class>
 				<owl:unionOf rdf:parseType="Collection">
-					<rdf:Description rdf:about="&iof-maint;DispositionToExhibitUndesirableBehaviour">
+					<rdf:Description rdf:about="&iof-maint;DispositionToExhibitUndesirableBehavior">
 					</rdf:Description>
 					<rdf:Description rdf:about="&iof-maint;DispositionToFail">
 					</rdf:Description>
 				</owl:unionOf>
 			</owl:Class>
 		</owl:equivalentClass>
 		<skos:example>disposition to leak, disposition to overheat, disposition to vibrate</skos:example>
-		<iof-av:firstOrderLogicDefinition>UndesirableDisposition(x) ↔ DispositionToFail(x) ∨ DispositionToExhibitUndesirableBehaviour</iof-av:firstOrderLogicDefinition>
+		<iof-av:firstOrderLogicDefinition>UndesirableDisposition(x) ↔ DispositionToFail(x) ∨ DispositionToExhibitUndesirableBehavior</iof-av:firstOrderLogicDefinition>
```

Comment:
I think this should read
`<iof-av:firstOrderLogicDefinition>UndesirableDisposition(x) ↔ DispositionToFail(x) ∨ DispositionToExhibitUndesirableBehavior(x)</iof-av:firstOrderLogicDefinition>`

Reply:
Hi I've made this edit and it will be addressed in a different pull request